opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -234,6 +234,20 @@ namespace Opm
 #endif
                 if ( unknownKeyWords )
                 {
+                    if ( output_cout_ )
+                    {
+                        std::string msg = std::string("Aborting simulation due unknown ")
+                            + "parameters. Please query \"flow --help\" for "
+                            + "supported command line parameters.";
```

Comment:
I think you can (and probably should) remove the concatenation operation before `"supported`.  You can concatenate string literals by placing them next to each other:
```C++
std::string msg = "Aborting ..."
    "parameter ..."
    "supported ...";
```

Reply:
Thanks. I did that and squashed.